src/screens/gm/tweak.h

```diff
@@ -199,6 +205,39 @@ class GuiShipTweakPlayer2 : public GuiTweakPage
     virtual void onDraw(sf::RenderTarget& window) override;
 };
 
+class GuiShipTweakCpu : public GuiTweakPage
+{
+private:
+    P<CpuShip> target;
+
+    GuiSlider* repair_rate_slider;
+public:
+    GuiShipTweakCpu(GuiContainer* owner);
+
+    virtual void open(P<SpaceObject> target) override;
+
+    virtual void onDraw(sf::RenderTarget& window) override;
+};
+
+class GuiStationTweakBase : public GuiTweakPage
```

Comment:
Reason there is "base" in this class name?

Reply:
Parallel to the base ship tweak class name